phpBB/styles/prosilver/theme/icons.css

```diff
@@ -85,3 +85,26 @@
     line-height: .9;
     height: 12px;
 }
+
+
+blockquote cite:before, .uncited:before {
+	-webkit-font-smoothing: antialiased;
+	-mos-osx-font-smoothing: grayscale;
+	font-family: FontAwesome;
+	font-size: 14px;
+	font-weight: normal;
+	font-style: normal;
+	font-variant: normal;
+	line-height: 1;
+	text-rendering: auto;
+	display: inline-block;
+	padding-right: 5px;
+	content: '\f10d'; /* Font Awesome quote-left */
+}
+
+.rtl blockquote cite:before, .rtl .uncited:before {
+	padding-right: 0;
+	padding-left: 5px;
+	content: '\f10e'; /* Font Awesome quote-right */
```

Comment:
Now I'm getting a bit confused (or lack the understanding) because within icons.css there are already other examples where padding is specified, so why shouldn't it also apply in the case?